src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationExtensions.cs

```diff
@@ -1,70 +1,77 @@
 ﻿/*
  * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
- * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * See https://LinkedIn.com/aspnet-contrib/AspNet.Security.OAuth.Providers
```

Comment:
This looks like it was changed by an over-aggressive Find-And-Replace.

Reply:
Good catch, I'll push a fix for this :)